src/frontend/src/pages/myfeeds.js

```diff
@@ -15,94 +34,312 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+function HelpPopoverButton() {
+  const classes = useStyles();
+
+  return (
+    <PopupState variant="popover" popupId="help-popover">
+      {(popupState) => (
+        <div>
+          <IconButton
+            color="secondary"
+            classes={{ root: classes.button }}
+            {...bindTrigger(popupState)}
+          >
+            <HelpOutline />
+          </IconButton>
+          <Popover
+            {...bindPopover(popupState)}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'center',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'center',
+            }}
+          >
+            <Box p={2}>
+              <Typography align="center">
+                Please enter the web address of your blog&apos;s RSS or Atom feed.
+              </Typography>
+              <Typography align="center">
+                <em>
+                  (<a href="https://rss.com/blog/find-rss-feed/">Not sure how to find that?</a>)
+                </em>
+              </Typography>
+            </Box>
+          </Popover>
+        </div>
+      )}
+    </PopupState>
+  );
+}
+
+function DeleteDialogButton({ feed }) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const removeFeed = () => {
+    console.log(`Removing feed hosted at URL ${feed.url}`);
+    // TODO
```

Comment:
File a follow-up issue for this, and put the URL to the issue here.

Reply:
Also, cc @c3ho and @Grommers00, who are working on backend bits for this.

Reply:
> File a follow-up issue for this, and put the URL to the issue here.

Done: https://github.com/Seneca-CDOT/telescope/issues/946
https://github.com/Seneca-CDOT/telescope/pull/933/commits/2ae7efe11944d53d23e43c897c561cf9948cfedb#diff-c1747a45c828916ed2e3274bdd5df4dcR35

> Also, cc @c3ho and @Grommers00, who are working on backend bits for this.

I'll definitely be in touch with Calvin and James via #946. I'm very curious to learn more about the My Feeds-related backend functionalities they've been working on.